src/v0/destinations/tune/transform.js

```diff
@@ -0,0 +1,90 @@
+const get = require('get-value');
+const { InstrumentationError } = require('@rudderstack/integrations-lib');
+const {
+  defaultRequestConfig,
+  simpleProcessRouterDest,
+  getHashFromArray,
+  isDefinedAndNotNull,
+  isNotEmpty,
+} = require('../../util');
+
+const mapPropertiesWithNestedSupport = (msg, properties, mappings) => {
+  const mappedObj = {}; // Create a new object for parameters
+  Object.entries(mappings).forEach(([key, value]) => {
+    const keyStr = `${key}`;
+    const args = { object: properties, key: keyStr };
+    if (key.split('.').length > 1) {
```

Comment:
`
```suggestion
    if (keyStr.split('.').length > 1) {
```

Reply:
It should be `args.key.split('.').length > 1`

Reply:
done